### PR TITLE
Quering multiple tables

### PR DESCRIPTION
**Pull request by Sboursen**
# Description

This is the second milestone of the Vet_clinic database.

## Milestone requirement:

- [x] Update the animal's table
- [x] Create owners Table
- [x] Create the species Table
- [x] Connect the tables using foreign keys 
- [x] Write the required queries

Thanks for taking the time to review this PR.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -81,3 +81,29 @@ GROUP BY species;
...
+
+SELECT animals.name AS animal 
+FROM animals
+JOIN species ON species_id = species.id
+WHERE species.name = 'Digimon' AND owner_id = (SELECT id FROM owners WHERE full_name = 'Jennifer Orwell');
+
+SELECT animals.name AS animal 
+FROM animals
+JOIN owners ON owner_id = owners.id
+WHERE animals.escape_attempts = 0 AND owner_id = (SELECT id FROM owners WHERE full_name = 'Dean Winchester');
```
Comment (Mihndim2020) [inline queries.sql:109]: - [x] I think are missing a query here, the query that returns the person who owns the most animals, as requested in the [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md), "`Who owns the most animals?`". Please kindly implement it.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @Sboursen, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Clean and organized code,
:heavy_check_mark: Descriptive Readme file.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (Sboursen) [reply queries.sql:109]: Hi @Mihndim2020 
Good catch, There were too many queries and I missed that one.
Thanks,

**Review by Lordkaito — APPROVED** [review]
Hi @Sboursen, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**